src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1515,7 +1517,7 @@ private static TypeSymbol GetParameterType(int argIndex, MemberAnalysisResult re
                     }
                 }
 
-                return BetterResult.Neither;
+                return PreferValOverInParameters(arguments, m1, m1LeastOverridenParameters, m2, m2LeastOverridenParameters);
```

Comment:
>return PreferValOverInParameters(arguments, m1, m1LeastOverridenParameters, m2, m2LeastOverridenParameters); [](start = 16, length = 108)

Do we have a test covering this code path?

Reply:
Yes. For example `PassingArgumentsToOverloadsOfByValAndInParameters_TypeConversions_NonConvertible`

---
In reply to: [155007343](https://github.com/dotnet/roslyn/pull/23122#discussion_r155007343) [](ancestors = 155007343)